### PR TITLE
Fix read binlog unsigned integer

### LINKED ISSUE #10314: shardingsphere-scaling transfer -1 to 255 for TINYINT when IN 'EXECUTE_INCREMENTAL_TASK'
## Bug Report
1.I use the master version of  shardingsphere-scaling to test database scaling.I have a table which has a column like this:
`client_type` TINYINT(2) NULL DEFAULT '-1';

2.when running shardingsphere-scaling transfer '-1'  value to '255' value.Then the sql will transfer like this：
insert into test(id,client_type) values(1,255)......;

3.I found that MySQLTinyBinlogProtocolValue.java use MySQLPacketPayload   readInt1() to read the value，but MySQLPacketPayload use ‘readUnsignedByte()’  to read value.

4.MySQLShortBinlogProtocolValue and other may have the same problem.

### PR DESCRIPTION
**Pull request by avalon5666**
Fixes #10314.

### CODE REVIEW

File: shardingsphere-scaling/shardingsphere-scaling-dialect/shardingsphere-scaling-mysql/src/main/java/org/apache/shardingsphere/scaling/mysql/component/column/metadata/MySQLColumnMetaData.java [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+import lombok.ToString;
+
+/**
+ * MySQL column meta data.
+ */
+@RequiredArgsConstructor
+@Getter
+@EqualsAndHashCode
+@ToString
+public class MySQLColumnMetaData {
```
Comment (tristaZero) [inline shardingsphere-scaling/shardingsphere-scaling-dialect/shardingsphere-scaling-mysql/src/main/java/org/apache/shardingsphere/scaling/mysql/component/column/metadata/MySQLColumnMetaData.java:32]: final one?

File: shardingsphere-scaling/shardingsphere-scaling-dialect/shardingsphere-scaling-mysql/src/test/java/org/apache/shardingsphere/scaling/mysql/component/column/value/UnsignedTinyintHandlerTest.java [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+package org.apache.shardingsphere.scaling.mysql.component.column.value;
+
+import org.junit.Test;
+
+import java.io.Serializable;
+
+import static org.hamcrest.CoreMatchers.is;
+import static org.junit.Assert.assertThat;
+
+public class UnsignedTinyintHandlerTest {
```
Comment (tristaZero) [inline shardingsphere-scaling/shardingsphere-scaling-dialect/shardingsphere-scaling-mysql/src/test/java/org/apache/shardingsphere/scaling/mysql/component/column/value/UnsignedTinyintHandlerTest.java:27]: final

File: shardingsphere-scaling/shardingsphere-scaling-dialect/shardingsphere-scaling-mysql/src/test/java/org/apache/shardingsphere/scaling/mysql/component/column/value/UnsignedSmallintHandlerTest.java [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+package org.apache.shardingsphere.scaling.mysql.component.column.value;
+
+import org.junit.Test;
+
+import java.io.Serializable;
+
+import static org.hamcrest.CoreMatchers.is;
+import static org.junit.Assert.assertThat;
+
+public class UnsignedSmallintHandlerTest {
```
Comment (tristaZero) [inline shardingsphere-scaling/shardingsphere-scaling-dialect/shardingsphere-scaling-mysql/src/test/java/org/apache/shardingsphere/scaling/mysql/component/column/value/UnsignedSmallintHandlerTest.java:27]: final

File: shardingsphere-scaling/shardingsphere-scaling-dialect/shardingsphere-scaling-mysql/src/test/java/org/apache/shardingsphere/scaling/mysql/component/column/value/UnsignedMediumintHandlerTest.java [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+package org.apache.shardingsphere.scaling.mysql.component.column.value;
+
+import org.junit.Test;
+
+import java.io.Serializable;
+
+import static org.hamcrest.CoreMatchers.is;
+import static org.junit.Assert.assertThat;
+
+public class UnsignedMediumintHandlerTest {
```
Comment (tristaZero) [inline shardingsphere-scaling/shardingsphere-scaling-dialect/shardingsphere-scaling-mysql/src/test/java/org/apache/shardingsphere/scaling/mysql/component/column/value/UnsignedMediumintHandlerTest.java:27]: final

File: shardingsphere-scaling/shardingsphere-scaling-dialect/shardingsphere-scaling-mysql/src/test/java/org/apache/shardingsphere/scaling/mysql/component/column/value/UnsignedIntHandlerTest.java [hunk 5]
```diff
@@ -0,0 +1,38 @@
...
+package org.apache.shardingsphere.scaling.mysql.component.column.value;
+
+import org.junit.Test;
+
+import java.io.Serializable;
+
+import static org.hamcrest.CoreMatchers.is;
+import static org.junit.Assert.assertThat;
+
+public class UnsignedIntHandlerTest {
```
Comment (tristaZero) [inline shardingsphere-scaling/shardingsphere-scaling-dialect/shardingsphere-scaling-mysql/src/test/java/org/apache/shardingsphere/scaling/mysql/component/column/value/UnsignedIntHandlerTest.java:27]: final

File: shardingsphere-scaling/shardingsphere-scaling-dialect/shardingsphere-scaling-mysql/src/test/java/org/apache/shardingsphere/scaling/mysql/component/column/value/UnsignedBigintHandlerTest.java [hunk 6]
```diff
@@ -0,0 +1,39 @@
...
+
+import org.junit.Test;
+
+import java.io.Serializable;
+import java.math.BigInteger;
+
+import static org.hamcrest.CoreMatchers.is;
+import static org.junit.Assert.assertThat;
+
+public class UnsignedBigintHandlerTest {
```
Comment (tristaZero) [inline shardingsphere-scaling/shardingsphere-scaling-dialect/shardingsphere-scaling-mysql/src/test/java/org/apache/shardingsphere/scaling/mysql/component/column/value/UnsignedBigintHandlerTest.java:28]: final

File: shardingsphere-scaling/shardingsphere-scaling-dialect/shardingsphere-scaling-mysql/src/main/resources/META-INF/services/org.apache.shardingsphere.scaling.mysql.component.column.value.ValueHandler [hunk 7]
```diff
@@ -0,0 +1,39 @@
...
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+org.apache.shardingsphere.scaling.mysql.component.column.value.UnsignedTinyintHandler
+org.apache.shardingsphere.scaling.mysql.component.column.value.UnsignedSmallintHandler
+org.apache.shardingsphere.scaling.mysql.component.column.value.UnsignedMediumintHandler
+org.apache.shardingsphere.scaling.mysql.component.column.value.UnsignedIntHandler
+org.apache.shardingsphere.scaling.mysql.component.column.value.UnsignedBigintHandler
```
Comment (tristaZero) [inline shardingsphere-scaling/shardingsphere-scaling-dialect/shardingsphere-scaling-mysql/src/main/resources/META-INF/services/org.apache.shardingsphere.scaling.mysql.component.column.value.ValueHandler:39]: It is better to add a new line below the last line.

File: shardingsphere-scaling/shardingsphere-scaling-dialect/shardingsphere-scaling-mysql/src/main/resources/META-INF/services/org.apache.shardingsphere.scaling.mysql.component.column.value.ValueHandler [hunk 8]
```diff
@@ -0,0 +1,39 @@
...
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+#
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
```
Comment (tristaZero) [inline shardingsphere-scaling/shardingsphere-scaling-dialect/shardingsphere-scaling-mysql/src/main/resources/META-INF/services/org.apache.shardingsphere.scaling.mysql.component.column.value.ValueHandler:20]: Hi, licenses are double.

File: shardingsphere-scaling/shardingsphere-scaling-dialect/shardingsphere-scaling-mysql/src/main/java/org/apache/shardingsphere/scaling/mysql/component/column/value/UnsignedBigintHandler.java [hunk 9]
```diff
@@ -0,0 +1,37 @@
...
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.shardingsphere.scaling.mysql.component.column.value;
+
+import java.io.Serializable;
+import java.math.BigInteger;
+
+public class UnsignedBigintHandler implements ValueHandler {
```
Comment (tristaZero) [inline shardingsphere-scaling/shardingsphere-scaling-dialect/shardingsphere-scaling-mysql/src/main/java/org/apache/shardingsphere/scaling/mysql/component/column/value/UnsignedBigintHandler.java:23]: final

**Review by tristaZero — CHANGES_REQUESTED** [review]
Cool @avalon5666 I am glad you can come back to take charge of sharding-scaling!

**Review by Lucas-307 — APPROVED** [review]

**Review by tristaZero — APPROVED** [review]

**Pull request merged**